.github/workflows/check-binary-size.yml

```diff
@@ -61,6 +61,7 @@ jobs:
         run: |
           # Bootstrap config
           cat <<EOF > config.toml
+          change-id = 9999999
```

Comment:
This doesn't seem to have worked? Maybe it's not seeing the config.toml file for some reason?

Reply:
It's also using actions/checkout@v3. It won't use the workflow file changes until they're merged.

Reply:
Oh I see! I get what you were saying now. This looks right to me so I guess we'll merge and see?